public/app/features/dashboard-scene/scene/layout-default/DefaultGridLayoutManager.tsx

```diff
@@ -402,12 +481,30 @@ export class DefaultGridLayoutManager
     return new DefaultGridLayoutManager({
       grid: new SceneGridLayout({
         children,
-        isDraggable: true,
-        isResizable: true,
+        isDraggable,
+        isResizable,
       }),
     });
   }
 
+  public handleVariableUpdateCompleted(variable: SceneVariable, hasChanged: boolean): void {
+    for (const child of this.state.grid.state.children) {
+      if (!(child instanceof SceneGridRow) || !child.state.$behaviors) {
+        continue;
+      }
+
+      for (const behavior of child.state.$behaviors) {
+        if (behavior instanceof RowRepeaterBehavior) {
+          if (behavior.isWaitingForVariables || (behavior.state.variableName === variable.state.name && hasChanged)) {
+            behavior.performRepeat(true);
+          } else if (!behavior.isWaitingForVariables && behavior.state.variableName === variable.state.name) {
+            behavior.notifyRepeatedPanelsWaitingForVariables(variable);
+          }
+        }
+      }
+    }
+  }
```

Comment:
After we merge https://github.com/grafana/grafana/pull/99432 we can remove this

Reply:
@bfmatei https://github.com/grafana/grafana/pull/99432 is now merged

Reply:
Rebased!